### DISCUSSION TITLE
Transaction confirmed 8 days ago, but the spendable balance is still at zero and the pending balance is the full amount

### DISCUSSION THREAD

**Discussion by SHexcoat** [Chia-Network/chia-blockchain#17162] [Support]
![Screenshot from 2023-12-24 20-55-30](https://github.com/Chia-Network/chia-blockchain/assets/117527850/9cb369d1-3813-45be-b9cf-cf0e504546f1)

**Comment by BrandtH22** [comment]
Hello @SHexcoat , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the pending balance, the current high activity of the chain means that transaction submitted without fees are given lower priority than those with fees. Since both of the transactions in the list are shown as not having fees it is very likely that they have not completed onchain because the fee pressure has been ongoing since about the 17th of December.

Unfortunately there is currently not a good method in the GUI to replace submitted transactions with a higher fee but what you can try is to stop chia, rename the wallet db at `\.chia\mainnet\wallet\db\`, start chia and let it sync, reattempt the previous offer accept but with a higher fee.

Let us know if you run into any issues or questions throughout the process 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
